storybook/decorators/with-wrapper.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * A Storybook decorator to wrap a story in a div applying a max width and
+ * padding. This can be used to simulate real world constraints on components
+ * such as being located within the WordPress editor sidebars.
+ */
+
+const Wrapper = styled.div`
+	max-width: 248px;
+	padding: 16px;
```

Comment:
Should we also show an `outline` ? It would help to read the padding better

Reply:
For sure there are pros and cons — I though an outline / border could help also with the look of the sidebar?

Reply:
A couple small things:

1. I believe the correct metrics are 280-(16*2) = 248px? (248px is the internal content width, after padding)
2. I'm hoping we can keep this add-on to have a singular purpose for now — so, just max-width and nothing else. (No padding, no borders, etc). Would that be ok?
3. One of the reasons for point 2 ☝️ is that almost every Storybook decorator we have is a "wrapper". So it would be good to be clear about what each wrapper is for. Could we change all the naming to be more specific, like `WithMaxWidthWrapper`, `context.globals.maxWidthWrapper`, etc?

We could also consider injecting an indicator div (like the margin checker) so nobody gets confused whether or not a max-width is being applied. No strong opinion on this at the moment though. We could add something later if we do find it confusing.

<img width="283" alt="Example of an indicator div to show that there's a max-width wrapper" src="https://user-images.githubusercontent.com/555336/196937373-18ffa9e8-51e5-4b35-972e-f284017e98b2.png">

Reply:
Thanks, all good suggestions. 

- Padding has been removed
- Max width has been corrected
- No border added
- Add-on has been renamed to indicate a singular purpose
- Indicator has also been added 
    - Styles copied from the margin checker
    - I think it's good to make it easy to know what max-width is being applied